lib/game.rb

```diff
@@ -0,0 +1,101 @@
+class Game
+  attr_reader :p1_board,
+              :cpu_board
+
+  def initialize()
+    @p1_board = Board.new
+    @p1_cruiser = Ship.new('cruiser', 3)
+    @p1_submarine = Ship.new('submarine', 2)
+
+    @cpu_board = Board.new
+    @cpu_cruiser = Ship.new('cruiser', 3)
+    @cpu_submarine = Ship.new('submarine', 2)
+  end
+
+  def start
+    p "Welcome to BATTLESHIP"
+    p "Enter p to play. Enter q to quit."
+    user_choice = gets.chomp
+    if user_choice == "q"
+      quit
+    else
+      board_setup
+    end
+  end
+
+  def quit
+    p "Bye bye!"
+  end
+
+  def board_setup
+    p "I have laid out my ships on the grid."
+    p "You now need to lay out your two ships and get ready to LOSE, sucker."
```

Comment:
what if we had just an array full of every line we wanted to print like this, and then we could just call on a specific index for a specific line or a range of indices for a "random" response to taking turns.